examples/detection_callback.py

```diff
@@ -24,6 +25,13 @@ def simple_callback(device: BLEDevice, advertisement_data: AdvertisementData):
 
 
 async def main(service_uuids):
+    if service_uuids[0] == "all":
```

Comment:
Won't this raise a `IndexError` if `service_uuids` is empty?

Reply:
yep you are right. Fixed.